src/fec_rs_gf2n_fft_add.h

```diff
@@ -215,6 +215,85 @@ class RsGf2nFftAdd : public FecCode<T> {
             output->set(i, S.get(i));
     }
 
+    // Lagrange interpolation w/ vector
+    void decode_vec_lagrange(
+        vec::Vector<T>* output,
+        const std::vector<Properties>& props,
+        off_t offset,
+        vec::Vector<T>* fragments_ids,
```

Comment:
Can some of these (`fragments_ids`, `words` and `vx`) be const reference? Or are they all modified in this function?

Reply:
Yes, they could be. But these APIs will be changed in very next PRs. So I will pay attention at that time :)